### PR TITLE
draft recipe for EDGE-pro

### PR DESCRIPTION
**Pull request by wjurkowski**
* [* ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ *] This PR adds a new recipe.
* [ *] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/edgepro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+build:
+  number: 0
+
+about:
+    home: http://ccb.jhu.edu/software/EDGE-pro/index.shtml
+    license: compliant with https://opensource.org/docs/definition.php
+    summary: EDGE-pro (Estimated Degree of Gene Expression in PROkaryotes) is an efficient software system to estimate gene expression levels in prokaryotic genomes from RNA-seq data. EDGE-pro uses Bowtie2 for alignment and then estimates expression directly from the alignment results. 
+
+EDGE-pro includes routines to assign reads aligning to overlapping gene regions accurately. 15% or more of bacterial genes overlap other genes, making this a significant problem for bacterial RNA-seq, one that is generally ignored by programs designed for eukaryotic RNA-seq experiments.
```
Comment (daler) [inline recipes/edgepro/meta.yaml]: @wjurkowski this line is probably causing the parsing error on travis.

File: recipes/edgepro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    fn: EDGE_pro_v1.3.1.tar.gz
+    sha256: 70fe107459703ba407d3555acfa9a64a8a0334a7
+
+requirements:
+  build:
+    - gcc
+  run:
+    - libgcc
+    - Perl
+    - Bowtie2 http://sourceforge.net/projects/bowtie-bio/files/bowtie2
```
Comment (nsoranzo) [inline recipes/edgepro/meta.yaml]:
```
    - perl
    - bowtie2
```

File: recipes/edgepro/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+build:
+  number: 0
+
+about:
+    home: http://ccb.jhu.edu/software/EDGE-pro/index.shtml
+    license: compliant with https://opensource.org/docs/definition.php
+    summary: EDGE-pro (Estimated Degree of Gene Expression in PROkaryotes) is an efficient software system to estimate gene expression levels in prokaryotic genomes from RNA-seq data. EDGE-pro uses Bowtie2 for alignment and then estimates expression directly from the alignment results. EDGE-pro includes routines to assign reads aligning to overlapping gene regions accurately. 15% or more of bacterial genes overlap other genes, making this a significant problem for bacterial RNA-seq, one that is generally ignored by programs designed for eukaryotic RNA-seq experiments. 
+
+package:
+    name: EDGE-pro
```
Comment (nsoranzo) [inline recipes/edgepro/meta.yaml]: This needs to be all lowercase

File: recipes/edgepro/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    fn: EDGE_pro_v1.3.1.tar.gz
+    sha256: 70fe107459703ba407d3555acfa9a64a8a0334a7
+
+requirements:
+  build:
+    - gcc
+  run:
+    - libgcc
+    - perl
+    - bowtie2 http://sourceforge.net/projects/bowtie-bio/files/bowtie2
```
Comment (nsoranzo) [inline recipes/edgepro/meta.yaml]: Can you remove the URL here please?

File: recipes/edgepro/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    license: compliant with https://opensource.org/docs/definition.php
+    summary: EDGE-pro (Estimated Degree of Gene Expression in PROkaryotes) is an efficient software system to estimate gene expression levels in prokaryotic genomes from RNA-seq data. EDGE-pro uses Bowtie2 for alignment and then estimates expression directly from the alignment results. EDGE-pro includes routines to assign reads aligning to overlapping gene regions accurately. 15% or more of bacterial genes overlap other genes, making this a significant problem for bacterial RNA-seq, one that is generally ignored by programs designed for eukaryotic RNA-seq experiments. 
+
+package:
+    name: edge-pro
+    version: 1.3.1
+source:
+    url: http://ccb.jhu.edu/software/EDGE-pro/EDGE_pro_v1.3.1.tar.gz
+    fn: EDGE_pro_v1.3.1.tar.gz
+    sha256: 70fe107459703ba407d3555acfa9a64a8a0334a7
```
Comment (nsoranzo) [inline recipes/edgepro/meta.yaml]: sha256 should be fe6e6bbd8a74294de25d6df1add76e2aa6df0d8df6e450b5905fa9bdbcae9e4e

Comment (wjurkowski) [reply recipes/edgepro/meta.yaml]: oh yes, I used wrong option. Thanks for spotting

**Comment by nsoranzo** [comment]
@wjurkowski I pushed to commits to your branch, let's see if it builds.

**Comment by nsoranzo** [comment]
The build failed because there is a missing "Switch" Perl module:
- a new `perl-switch` package needs to be added to conda-forge, there is already a stalled PR adding it (together with other packages): https://github.com/conda-forge/staged-recipes/pull/3261/files#diff-db994ac164610b682f130bd46ddfec3a
- `perl-switch` needs to be added to the `edge-pro` run requirements.

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**